tutorials/3d/3d_rendering_limitations.rst

```diff
@@ -45,11 +45,12 @@ There are two main ways to alleviate banding:
   fullscreen debanding shader as a post-processing effect and is very cheap.
   Fullscreen debanding is only supported when using the GLES3 or Vulkan renderers.
```

Comment:
The reference to GLES3 here (and above) should be removed. The OpenGL renderer does not render in HDR and debanding isn't available yet